packages/contracts-bedrock/test/DeployOPChain.t.sol

```diff
@@ -399,6 +410,15 @@ contract DeployOPChain_Test is DeployOPChain_TestBase {
         basefeeScalar = uint32(uint256(hash(_seed, 6)));
         blobBaseFeeScalar = uint32(uint256(hash(_seed, 7)));
         l2ChainId = uint256(uint256(hash(_seed, 8)));
+        uint256 numStartingAnchorRoots = bound(uint256(uint256(hash(_seed, 9))), 1, 10);
+        for (uint256 i = 0; i < numStartingAnchorRoots; i++) {
+            startingAnchorRoots.push(
+                AnchorStateRegistry.StartingAnchorRoot({
+                    gameType: GameTypes.CANNON,
+                    outputRoot: OutputRoot({ root: Hash.wrap(keccak256(abi.encode(_seed, i))), l2BlockNumber: 400 + i })
```

Comment:
For my own understanding: is having each root incremented by 1 block a requirement, or it's arbitrary and we just increment by 1 each loop iteration for testing?

Reply:
It was arbitrary, but I've improved this section to be more true to real world usage.